AppCreationScripts/apps.json

```diff
@@ -0,0 +1,33 @@
+{
+  "Sample": {
+    "Title": "Calling Microsoft Graph from an Android app",
+    "Level": 200,
+    "Client": "Android"
+  },
+  "AppRegistrations": [
+    {
+      "x-ms-id": "active-directory-android-java",
+      "x-ms-name": "android-java",
+      "x-ms-version": "1.0",
+      "replyUrlsWithType": [
+        {
+          "url": "msauth://{package_name}/{signature_hash}",
```

Comment:
I'm not seeing package_name and signature_hash in the document, so it's better to get someone in the portal team to review this before merging this PR.

Otherwise, lgtm.

Reply:
Use these tokens instead, "Enter_the_Package_Name" and "Enter_the_Signature_Hash"

Reply:
@hamiltonha oopsie. We might need to create a new PR for this?